components/topic/RelatedContent.tsx

```diff
@@ -0,0 +1,117 @@
+import Link from "next/link";
```

Comment:
Yes, the way RelatedContent is built right now is that it assumes there'll be 6 related topics. The reason it was built this was was because I was having a very hard time making the bootstrap carousel adaptable to different screen sizes, so right now its hardcoded with 6 links. I probably will need help finding a better solution because for example, showing four cards on a small screen doesn't look good but I'm just not sure of a good solution to it using the bootstrap carousel.